### ISSUE TITLE
`Method to resolve Ethereum Name Service (ENS)` dropdown items are not translated in other locales

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#14864]
## Description 

`Method to resolve Ethereum Name Service (ENS)` dropdown items are not translated in other locales

## Steps to Reproduce

   1. Set OS locale or France or Germany or other locales
   2. Open brave 1.22.66
   3. Open brave://settings/extensions
   4. Look for `Method to resolve Ethereum Name Service (ENS)` dropdown items

## Actual result:

![image](https://user-images.githubusercontent.com/38657976/111981884-4b1ed380-8b2e-11eb-9755-3a9e7ead0d4e.png)

## Expected result:

Dropdown items should be translated in other locales
## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 1.22.66 Chromium: 89.0.4389.90 (Build officiel) (64 bits)
-- | --
Révision | 62eb262cdaae9ef819aadd778193781455ec7a49-refs/branch-heads/4389@{#1534}
Système d'exploitation | Windows 10 OS Version 2004 (Build 19041.867)

## Version/Channel Information:

- Can you reproduce this issue with the current release? NA
- Can you reproduce this issue with the beta channel? NA
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

cc: @brave/legacy_qa @mkarolin @rebron

**Comment by mkarolin** [comment]
cc: @yrliou

**Comment by LaurenWags** [comment]
Reproduces on Nightly (using Spanish) as well

```

Brave | 1.24.10 Chromium: 89.0.4389.90 (Build oficial) nightly (x86_64)
-- | --
Revisión | 62eb262cdaae9ef819aadd778193781455ec7a49-refs/branch-heads/4389@{#1534}
Sistema operativo | macOS Versión 10.15.7 (Compilación 19H524)

```

<img width="716" alt="Captura de pantalla 2021-03-22 a las 9 45 01 a  m" src="https://user-images.githubusercontent.com/28145373/111999369-4983ea00-8af3-11eb-94b1-e3b8293174d0.png">

**Comment by kjozwiak** [comment]
Added `QA/Blocked` as we'll need to wait for the translations to get merged into `1.23.x`. @mkarolin please keep us updated re: merging translations into `1.23.x` and unblocking the above 👍

**Comment by stephendonner** [comment]
Removing `QA/Blocked` label, as we now have the `1.23.70` build with updated translations.

Verified `PASSED` using 

Brave | 1.23.70 Chromium: 90.0.4430.70 (Official Build) (x86_64)
-- | --
Revision | 3954de7175366f3b7edca576f140dfa273e6b5ae-refs/branch-heads/4430@{#1210}
OS | macOS Version 11.2.3 (Build 20D91)

Confirmed using the inline testplan that -- in both French and Spanish -- the dropdown options for `Method to resolve Ethereum Name Service (ENS)` have been translated.

French | Spanish
-------|---------
<img width="843" alt="Capture d’écran 2021-04-13 à 15 11 31" src="https://user-images.githubusercontent.com/387249/114628250-6da69700-9c6b-11eb-9c6b-3cc665fb5885.png"> | <img width="845" alt="Captura de pantalla 2021-04-13 a las 15 13 41" src="https://user-images.githubusercontent.com/387249/114628260-6ed7c400-9c6b-11eb-930f-4d77bd22ba7e.png">
